graphql/types/Item.graphql

```diff
@@ -43,6 +43,7 @@ type Item {
   skuSpecifications: [SKUSpecification!]!
   uniqueId: String!
   unitMultiplier: Float
+  priceTags: [PriceTags]
```

Comment:
```suggestion
  priceTags: [PriceTags!]!
```

Reply:
@lucasecdb setting `priceTags` as required can't be dangerous for websites that uses the `orderForm` query in production?

Reply:
doesn't it always return an empty array? and this won't affect any stores unless we update the orderForm query in checkout-resources

Reply:
@lucasecdb actually we have another PR that adds `priceTags` on `ItemFragment` in checkout-resources.
https://github.com/vtex-apps/checkout-resources/pull/58

Reply:
oh, can you name the type in singular? like `PriceTag`

Reply:
@lucasecdb all suggestions was applied :)